From 109312fa9789b1129b75b173501907904ff3a0c6 Mon Sep 17 00:00:00 2001
From: balajigudala2368 <144100471+balajigudala2368@users.noreply.github.com>
Date: Thu, 7 Sep 2023 11:34:18 +0000
Subject: [PATCH] first hello

Signed-off-by: balajigudala2368 <144100471+balajigudala2368@users.noreply.github.com>
---
 file1.c | 9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 file1.c

diff --git a/file1.c b/file1.c
new file mode 100644
index 0000000..0f94834
--- /dev/null
+++ b/file1.c
@@ -0,0 +1,9 @@
+#include <stdio.h>
+#include <unistd.h>
+
+int main() {
+	printf("\nhello guys\nlkjfdhbkl");
+	sleep(5);
+	printf("\rI'm new to github\n");
+	return 0;
+}
-- 
2.41.0

